src/coreclr/vm/methodtable.cpp

```diff
@@ -2190,6 +2190,47 @@ BOOL MethodTable::IsClassPreInited()
 
 //========================================================================================
 
+namespace
+{
+    // Does this type have fields that are implicitly defined through repetition and not explicitly defined in metadata?
+    bool HasImpliedRepeatedFields(MethodTable* pMT, MethodTable* pFirstFieldValueType = nullptr)
+    {
+        CONTRACTL
+        {
+            THROWS;
+            GC_TRIGGERS;
+            MODE_ANY;
```

Comment:
```suggestion
            STANDARD_VM_CHECK;
```
Can it be just this? Or is this actually called in both GC modes?

Reply:
The caller is marked `MODE_ANY`, so I decided to copy it. I'll go trace through the callers and see if both modes are used.